src/main/python/camdkit/model.py

```diff
@@ -117,6 +114,7 @@ class FStop(StrictlyPositiveIntegerParameter):
   canonical_name = "fStop"
   sampling = Sampling.REGULAR
   units = "0.001 unit"
+  section = "lens"
```

Comment:
Why is the only parameter moved to `lens`?

Reply:
More will come as I implement the rest of the model